better quota validation

individual-it opened the pull request:
this fixes #3634

1. fixed computerFileSize to be more picky about incorrect values
2. more tests for computerFileSize
3. use computerFileSize to validate user quota

individual-it committed:
better quota validation

this fixes #3634

1. fixed computerFileSize to be more picky about incorrect values
2. more tests for computerFileSize
3. use computerFileSize to validate user quota

Signed-off-by: Artur Neumann <info@individual-it.net>

rullzer commented:
@individual-it to merge your changes we require you to sign off on your commits. Could you do that?

MorrisJobke commented:
> @individual-it to merge your changes we require you to sign off on your commits. Could you do that?

For details see https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md#sign-your-work

individual-it commented:
squashed it all into the original commit that was already signed off

individual-it committed:
make values with white spaces possible

a user might enter values with white spaces, and that should be possible
and valid

Signed-off-by: Artur Neumann <info@individual-it.net>
